ci-operator/step-registry/gather/alibabacloud-console/gather-alibabacloud-console-commands.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+export ALIBABA_CLOUD_CREDENTIALS_FILE=${SHARED_DIR}/inicreds
+export ALIBABA_CLI_CREDENTIALS_FILE=${SHARED_DIR}/config
+
+wget https://aliyuncli.alicdn.com/aliyun-cli-linux-latest-amd64.tgz -O /tmp/aliyun-cli.tgz
+pushd /tmp
+tar zxvf aliyun-cli.tgz
+popd
+
+REGION="${LEASED_RESOURCE}"
+/tmp/aliyun version
+
+echo "Settting --config-path=${ALIBABA_CLI_CREDENTIALS_FILE} and --region=${LEASED_RESOURCE}"
+/tmp/aliyun  --config-path ${ALIBABA_CLI_CREDENTIALS_FILE} configure set --region "${LEASED_RESOURCE}"
```

Comment:
nit: missing quotes here and around some other variable expansions in this file.  Unlikely to be whitespace in the `ALIBABA_CLI_CREDENTIALS_FILE` value, but quoting the expansion means the reader doesn't have to wonder about that chance.

Reply:
Will update with quotes.